testing/ssh-tools/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Sven Wick <sven.wick@gmx.de>
+# Maintainer: Sven Wick <sven.wick@gmx.de>
+
+pkgname=ssh-tools
+pkgver=1.5
+pkgrel=0
+pkgdesc="collection of various tools using ssh"
+arch="noarch"
+url="https://github.com/vaporup/ssh-tools/"
+license="GPL-3.0"
```

Comment:
GPL-3.0 is ambiguous and not SPDX compliant, you are missing the -or-later or -only suffix